docs/Readme.md

```diff
@@ -15,15 +15,17 @@ Demo
 For live examples see our demo pages:
 
 *   [Highcharts demo](https://highcharts.com/demo/)
-*   [Highstock demo](https://highcharts.com/stock/demo/)
-*   [Highmaps demo](https://highcharts.com/maps/demo/)
+*   [Highcharts Stock demo](https://highcharts.com/stock/demo/)
+*   [Highcharts Maps demo](https://highcharts.com/maps/demo/)
+*   [Highcharts Gantt demo](https://highcharts.com/gantt/demo/)
 
 API
 ---
 
-For more specific information on Highcharts and Highstock options and functions, visit our API sites which also include several live and customizeable examples.
+For more specific information on Highcharts and Highcharts Stock options and functions, visit our API sites which also include several live and customizeable examples.
```

Comment:
We can rephrase this to "... information on Highcharts options and functions ..."

Since the Highcharts name now includes Stock, Maps and Gantt, plus the main packages are listed below.